packages/webdriverio/src/utils/implicitWait.js

```diff
@@ -24,6 +23,22 @@ export default async function implicitWait (currentElement, commandName) {
              */
             return await currentElement.parent.$(currentElement.selector)
         } catch {
+
+            if (currentElement.selector.toString().includes('this.previousElementSibling')) {
+                throw new Error(
+                    `Can't call ${commandName} on previous element of element with selector "${currentElement.parent.selector}" because sibling wasn't found`)
+            }
+
+            if (currentElement.selector.toString().includes('this.nextElementSibling')) {
+                throw new Error(
+                    `Can't call ${commandName} on next element of element with selector "${currentElement.parent.selector}" because sibling wasn't found`)
```

Comment:
changed to 'because sibling wasn't found' as I thought it reads better in error message and gives more of a clue to what has gone wrong in test.